benchexec/container.py

```diff
@@ -335,13 +337,30 @@ def setup_user_mapping(
         # but if it does not exist, we do not need to write to it.
         if e.errno != errno.ENOENT:
             logging.warning("Could not write to setgroups file in /proc: %s", e)
+            exception_occurred = True
 
     try:
         # map gid internally to our gid externally
         gid_map = f"{gid} {parent_gid} 1"
         util.write_file(gid_map, proc_child, "gid_map")
     except OSError as e:
         logging.warning("Creating GID mapping into container failed: %s", e)
+        exception_occurred = True
+
+    # Ubuntu 24.04 (and possibly later versions) restricts user namespaces,
+    # output error message here
+    if (
+        exception_occurred
+        and util.try_read_file("/proc/sys/kernel/apparmor_restrict_unprivileged_userns")
+        == "1"
+    ):
+        logging.warning(
+            "Ubuntu 24.04 restircts user namespaces, "
```

Comment:
```suggestion
            "Ubuntu 24.04 restricts user namespaces, "
```
(also in other file)

Reply:
I will fix this typo in a later commit. Thanks